### PR TITLE
changes some plot styles for the polar view

### PR DESCRIPTION
**Pull request by saransh13**
Example polar view plot below. In the long term, we should make the plot styles editable in the GUI, but for now style this seems a little better to me.

Screenshot 2024-11-04 at 3 29 31 PM

### CODE REVIEW

File: hexrdgui/image_canvas.py [hunk 1]
```diff
@@ -1034,10 +1037,39 @@ def finish_show_polar(self, iviewer):
                 }
                 self.axes_images.append(self.axis.imshow(**kwargs))
                 self.axis.axis('auto')
+
+                self.axis.yaxis.set_major_locator(AutoLocator())
+                self.axis.yaxis.set_minor_locator(AutoMinorLocator())
+
+                self.axis.xaxis.set_major_locator(AutoLocator())
+                self.axis.xaxis.set_minor_locator(AutoMinorLocator())
```
Comment (psavery) [inline hexrdgui/image_canvas.py:1045]: I think we can use the `PolarXAxisTickLocator` to fix the major locator for when `Q` is the x-axis type. However, the minor locator will need to either be removed, or a custom minor locator will need to be made, for when `Q` is the x-axis type.

File: hexrdgui/image_canvas.py [hunk 2]
```diff
@@ -35,6 +35,8 @@
 )
 from hexrdgui.utils.tth_distortion import apply_tth_distortion_if_needed
 
+FONTSIZE_LABEL=15
+FONTSIZE_TICKS=15
```
[the remark this answers is not in the document]
Comment (saransh13) [reply hexrdgui/image_canvas.py:39]: I like the idea of `4 + global_font_size`!

File: hexrdgui/image_canvas.py [hunk 3]
```diff
@@ -1087,13 +1170,14 @@ def finish_show_polar(self, iviewer):
                     'interpolation': 'none',
                 }
                 self.axes_images.append(self.axis.imshow(**kwargs))
-                self.axis.set_ylabel(r'$\eta$ [deg]')
+                self.axis.set_ylabel(r'$\phi$ [deg]', fontsize=FONTSIZE_LABEL, family='serif')
```
[the remark this answers is not in the document]
Comment (saransh13) [reply hexrdgui/image_canvas.py:1169]: Yes, I agree. let's change this back to `eta` and make it editable in a separate PR.

File: hexrdgui/image_canvas.py (hunk 1 above)
Comment (psavery) [reply hexrdgui/image_canvas.py]: We'll get it in this PR.

**Comment by ChristosT** [comment]
major and minor ticks using the new class:

Screenshot from 2024-11-15 15-55-33

File: hexrdgui/image_canvas.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (psavery) [reply hexrdgui/image_canvas.py]: Done.

File: hexrdgui/image_canvas.py (hunk 1 above)
Comment (psavery) [reply hexrdgui/image_canvas.py]: It's done. Christos added a commit that fixed it.

File: hexrdgui/image_canvas.py [hunk 4]
```diff
@@ -1034,10 +1039,39 @@ def finish_show_polar(self, iviewer):
...
+                self.axis.tick_params(**kwargs)
+
+                kwargs['which'] = 'minor'
+                kwargs['length'] = 2
+                self.axis.tick_params(**kwargs)
+
+                self.axis.tick_params(
+                   bottom=True, top=True, which='major', length=8)
+                self.axis.tick_params(
+                    bottom=True, top=True, which='minor', length=2)
```
Comment (psavery) [inline hexrdgui/image_canvas.py:1068]: I believe these don't have have an effect (these appear to be set by the previous calls to `tick_params`, and I don't see a difference in the plots if I remove these lines).

Do you know if these lines are necessary?

**Review by psavery — APPROVED** [review]
@saransh13 Try out the latest changes. If it looks good to you, then this is ready to be merged!

File: hexrdgui/image_canvas.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (psavery) [reply hexrdgui/image_canvas.py]: Done!

**Pull request merged**